grafana-plugin/src/models/user/user.types.ts

```diff
@@ -43,6 +46,14 @@ export interface User {
   hidden_fields?: boolean;
   timezone: Timezone;
   working_hours: { [key: string]: [] };
-  is_currently_oncall?: boolean;
-  teams?: GrafanaTeam[];
+}
+
+export interface PagedUser extends BaseUser {
+  important: boolean;
+}
+
+export interface UserCurrentlyOnCall extends BaseUser {
+  timezone: Timezone;
+  is_currently_oncall: boolean;
+  teams: GrafanaTeam[];
 }
```

Comment:
this is the response schema that the internal API returns when you specify the `is_currently_oncall` query param to `GET /users` (which is what the add responders popup uses)